Update to netstandard2.1;net6.0;net8.0 sunset .net framework

krzkraw opened the pull request:
I've done some upgrading to .netstandard 2.1 /.net6 /.net8 and rules project files to new formats to facilitate that. Unit tests pass in VS runner and nCrunch of course.

If this is more of an archive repo (no merges) then treat this as sticky note that I've forked and updated this. Thanks for posting this repo - it might save me loaaads of work.

DavidSdot committed:
Escaped escape char before segment delimiter

Current check only checks for an escaped segment delimiter. If the escape char itself is escaped the segment delimiter ist not recognized. ex: RFF+ON:??!

krzkraw committed:
Update to netstandard2.1;net6.0;net8.0 sunset .net framework

krzkraw committed:
Merge pull request #1 from DavidSdot/master

Escaped escape char before segment delimiter

Silvenga commented:
Thanks for the PR, and yes, we are basically in archive-mode, as neither myself nor @gamingrobot are currently in the healthcare industry.

Would you be willing to receive contributor roles in this repository?

krzkraw committed:
Auto refactors

pragma warning disable CA2021 in unit tests
Inline out var declarations
Use primary constructors
CA1829 Enumerable.Count() potentially enumerates the sequence while a Length/Count property is a direct access
inline out var

krzkraw committed:
Update nugets

krzkraw committed:
EF_X12_004010_TS850 EDIFACT_D96A

krzkraw committed:
Add common reader interface
